### PR TITLE
Tests for border control rules

### PR DESCRIPTION
**Pull request by NC-007**

### CODE REVIEW

File: FHICORC.Tests/BusinessRules/BorderControlRulesTests.cs [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+                {
+                    atLeastOneFalse = true;
+                    break;
+                }
+            }
+            Assert.AreEqual(true, atLeastOneFalse);
+        }
+
+        [Test]
+        public void Recovery_AfterValidPeriod_UnknownType_RedResult()
```
Comment (mbnnetcompany) [inline FHICORC.Tests/BusinessRules/BorderControlRulesTests.cs:473]: We do not get the type of test used for recovery, it is not part of the data. We should only test on the period. Change name of test to reflect this.

File: FHICORC.Tests/BusinessRules/BorderControlRulesTests.cs [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+                {
+                    atLeastOneFalse = true;
+                    break;
+                }
+            }
+            Assert.AreEqual(true, atLeastOneFalse);
+        }
+
+        [Test]
+        public void Recovery_BeforeValidPeriod_UnknownType_RedResult()
```
Comment (mbnnetcompany) [inline FHICORC.Tests/BusinessRules/BorderControlRulesTests.cs:452]: We do not get the type of test used for recovery, it is not part of the data. We should only test on the period. Change name of test to reflect this.

File: FHICORC.Tests/BusinessRules/BorderControlRulesTests.cs [hunk 3]
```diff
@@ -0,0 +1,547 @@
...
+            var rules = ruleSelectorService.SelectRules(dccPayload);
+            var results = ruleVerifierService.Verify(rules, rulesModel);
+            foreach (var result in results)
+            {
+                Assert.AreEqual(RulesFeedbackResult.FALSE, result.Result);
+            }
+        }
+
+        [Test]
+        public void Recovery_InValidPeriod_UnknownType_RedResult()
```
Comment (mbnnetcompany) [inline FHICORC.Tests/BusinessRules/BorderControlRulesTests.cs:432]: We do not get the type of test used for recovery, it is not part of the data. We should only test on the period. Change name of test to reflect this.

File: FHICORC.Tests/BusinessRules/BorderControlRulesTests.cs (hunk 1 above)
Comment (NC-007) [reply FHICORC.Tests/BusinessRules/BorderControlRulesTests.cs]: This is covered by another test then. Deleted this one

File: FHICORC.Tests/BusinessRules/BorderControlRulesTests.cs (hunk 2 above)
Comment (NC-007) [reply FHICORC.Tests/BusinessRules/BorderControlRulesTests.cs]: This is covered by another test then. Deleted this one

File: FHICORC.Tests/BusinessRules/BorderControlRulesTests.cs (hunk 3 above)
Comment (NC-007) [reply FHICORC.Tests/BusinessRules/BorderControlRulesTests.cs]: This is covered by another test then. Deleted this one

**Review by NC-002 — APPROVED** [review]

**Pull request merged**